docs/api.rst

```diff
@@ -186,7 +186,7 @@ Handling user creation, sign-in, log-out and querying
    :reqjson string password: The password that unlocks the account
    :reqjson bool sync_approval: A string denoting if the user approved an initial syncing of data from premium. Valid values are ``"unknown"``, ``"yes"`` and ``"no"``. Should always be ``"unknown"`` at first and only if the user approves should a login with approval as ``"yes`` be sent. If he does not approve a login with approval as ``"no"`` should be sent. If there is the possibility of data sync from the premium server and this is ``"unknown"`` the login will fail with an appropriate error asking the consumer of the api to set it to ``"yes"`` or ``"no"``.
 
-   **Example Response**:
```

Comment:
Why change this? Do not change this in only one endpoint. Makes absolutely no sense to break the api endpoint template. Either change the example response or add a note after it.